src/components/Navigation.js

```diff
@@ -165,10 +165,10 @@ function Navigation({ history }) {
               </div>
             </li>
           </ul>
-          <div className="navbar-right d-flex flex-row align-items-center navigation-search">
+          <div className="d-flex flex-row align-items-center ms-auto navigation-search">
```

Comment:
`navbar-right` not needed anymore?

Reply:
Not anymore, this behavior is now by default ( [link](https://getbootstrap.com/docs/5.0/components/navbar/#toggler) )

I just added an utility for "margin-right: automatic" so that the search box would stay on the right.